samples/sgx-ra/README.md

```diff
@@ -37,6 +37,15 @@ $ echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bioni
 $ wget -O - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
 $ sudo apt-get update
 $ sudo apt-get install -y libsgx-uae-service libsgx-dcap-default-qpl-dev libsgx-dcap-ql-dev libsgx-dcap-quote-verify-dev
+$ # install SGX SSL Library
+$ git clone https://github.com/intel/intel-sgx-ssl.git
+$ wget https://www.openssl.org/source/openssl-1.1.1q.tar.gz
+$ cp openssl-1.1.1q.tar.gz intel-sgx-ssl/openssl_source
+$ rm -f openssl-1.1.1q.tar.gz
+$ cd intel-sgx-ssl/Linux
+$ source /opt/intel/sgxsdk/environment
+$ make all
```

Comment:
When executing the `make all` command after having installed Intel SGX SSL, I have got the following error:

> Please follow the instruction at https://github.com/intel/linux-sgx/blob/master/README.md#build-the-intelr-sgx-sdk-and-intelr-sgx-psw-package to install "mitigation tools" first.
> Makefile:41: recipe for target 'sgxssl' failed
> make: *** [sgxssl] Error 1

Note that I have installed Intel SGX software using the package manager of Ubuntu (18.04), which I think is pretty standard. To have this issue fixed, I needed to execute in addition the following commands:

```shell
$ git clone https://github.com/intel/linux-sgx.git
$ cd linux-sgx && make preparation
$ sudo cp external/toolset/{current_distr}/* /usr/local/bin
# Verify that the paths are correctly set 
$ which ar as ld objcopy objdump ranlib
```

I think we should explicitly indicate these steps in the README in case this is not done beforehand.